scene/2d/camera_2d.cpp

```diff
@@ -33,6 +33,11 @@
 #include "core/config/project_settings.h"
 #include "scene/main/window.h"
 
+
```

Comment:
Need to remove this line for style, multiple empty lines are not permitted

Recommend using code formatting, see [here](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#doc-code-style-guidelines)